api/resources/brevets.py

```diff
@@ -27,3 +28,15 @@
 # it from a MongoEngine query object to a JSON and send back the JSON
 # directly instead of letting Flask-RESTful attempt to convert it to a
 # JSON for you.
+
+class BrevetsName(Resource):
+    
+    def get(self):
+        brevets = Brevet.objects().to_json()
+        return Response(brevets, mimetype="application/json", status=200)
+    
+    def post(self):
+        input_json = request.json
+        brevet = Brevet(**input_json).save()
+        ret_val = {"id": str(brevet.id)}
+        return ret_val, 200
```

Comment:
```suggestion
        return {"id": str(brevet.id)}, 200
```
Same thing but shrinks the explanation of your code just in case someone is curious what ret_val is. Great work though!